### PR TITLE
Vue Guide for Storybook v6

### PR DESCRIPTION
**Pull request by andre-brdoch**
Hey! I updated the Vue guide to storybook v6 😃 

Snippets can be tested here: https://github.com/andre-brdoch/learnstorybook-v6-vue
@jonniebigodes already reviewed the snippets there 😄 

I noticed that some chapters like the one for testing or deploys were quite different in the React guide. I assumed the React guide is most up to date, so i adapted those for Vue. Let me know if that was not wanted!

I added the snippets only to the english version, since it probably makes more sense to update snippets and texts simultanously for the other languages. Have a good day 👍

### CODE REVIEW

File: content/intro-to-storybook/vue/en/composite-component.md [hunk 1]
```diff
@@ -248,21 +210,20 @@ However, sometimes the devil is in the details. A test framework that is explici
...
 
 it('renders pinned tasks at the start of the list', () => {
-  const Constructor = Vue.extend(TaskList);
+  // render WithPinnedTasks story
+  const Constructor = Vue.extend(WithPinnedTasks);
   const vm = new Constructor({
-    propsData: { tasks: withPinnedTasksData },
+    propsData: WithPinnedTasks.args,
   }).$mount();
   const firstTaskPinned = vm.$el.querySelector('.list-item:nth-child(1).TASK_PINNED');
```
Comment (jonniebigodes) [inline content/intro-to-storybook/vue/en/composite-component.md:229]: @andre-brdoch when you're able update the wording to :
"Create a test file called `tests/unit/TaskList.spec.js`. Here we’ll build out our tests that make assertions about the output".

And the snippet should be:
```js
// tests/unit/TaskList.spec.js

import Vue from "vue";

import TaskList from "../../src/components/TaskList.vue";

import { WithPinnedTasks } from "../../src/components/TaskList.stories";

it("renders pinned tasks at the start of the list", () => {
  const Constructor = Vue.extend(TaskList); 
  const vm = new Constructor({
    propsData: WithPinnedTasks.args,
  }).$mount();
  const firstTaskPinned = vm.$el.querySelector(
    ".list-item:nth-child(1).TASK_PINNED"
  );
  // We expect the pinned task to be rendered first, not at the end
  expect(firstTaskPinned).not.toBe(null);
});

```

In React it's good to go and we could import directly the story and the associated template and the test would run fine. With Vue it's a bit different. We'll need to make a small compromise and inject the component directly. See the [issue](https://github.com/andre-brdoch/learnstorybook-v6-vue/issues/3) i raised in your repo for more context.

File: content/intro-to-storybook/vue/en/data.md [hunk 2]
```diff
@@ -214,21 +196,20 @@ export const Empty = () => ({
...
   const vm = new Constructor({
-    propsData: { tasks: withPinnedTasksData },
+    propsData: WithPinnedTasks.args,
   }).$mount();
-  const lastTaskInput = vm.$el.querySelector('.list-item:nth-child(1).TASK_PINNED');
+  const firstTaskPinned = vm.$el.querySelector('.list-item:nth-child(1).TASK_PINNED');
 
   // We expect the pinned task to be rendered first, not at the end
-  expect(lastTaskInput).not.toBe(null);
+  expect(firstTaskPinned).not.toBe(null);
```
Comment (jonniebigodes) [inline content/intro-to-storybook/vue/en/data.md:212]: @andre-brdoch see my previous comment regarding the test and associated [issue](https://github.com/andre-brdoch/learnstorybook-v6-vue/issues/3) for more details. 

Let me know when you've made the changes and we'll get this one merged.

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@andre-brdoch left two items that need some work. Other than that we're good to go. Can't thank you enough for this. Really stoked to get this one merged and shared with the community. Awesome job you did here :)

**Comment by andre-brdoch** [comment]
Hi @jonniebigodes, thank you so much for pointing this out! Totally missed that I was not actually running the tests 🙃 Thanks for letting me know, this is great to know for the future. I applied the fixes, let me know if there is anything left to do 😃 

Have a good one!

File: content/intro-to-storybook/vue/en/deploy.md [hunk 3]
```diff
@@ -1,99 +1,155 @@
...
 ```bash
 $ git remote add origin https://github.com/<your username>/taskbox.git
 ```
 
-Finally push the repo to GitHub
+Finally, push our local repo to the remote repo on GitHub with:
 
 ```bash
 $ git push -u origin master
 ```
```
Comment (jonniebigodes) [inline content/intro-to-storybook/vue/en/deploy.md:49]: @andre-brdoch replace master with main. GitHub recently updated the branch naming policy and with that the tutorials were updated as well to reflect this

File: content/intro-to-storybook/vue/en/deploy.md [hunk 4]
```diff
@@ -1,99 +1,155 @@
...
+
+```bash
+git commit -m "GitHub action setup"
+```
+
+Finally push them to the remote repository with:
+
+```bash
+git push origin master
+```
```
Comment (jonniebigodes) [inline content/intro-to-storybook/vue/en/deploy.md:139]: @andre-brdoch replace master with main in here as well so that it matches the above.

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@andre-brdoch when you're able address the two items i pointed out regarding the name of the branches and we'll be good to go. 

Let me know once you've applied them and i'll get this merged.

Stay safe

**Comment by andre-brdoch** [comment]
@jonniebigodes done! 👍

**Comment by jonniebigodes** [comment]
@andre-brdoch going to merge it as is. Thank you for this much deserved update to the tutorial. We really appreciate it. 👍 

Stay safe.

**Pull request merged**